### ISSUE TITLE
Displaying layers in the proper order, so all are active

### ISSUE THREAD

**Issue by DBaranskiW** [ushahidi/Ushahidi_Web#1236]
We are a self hosted Ushahidi deployment, http://candlevoice.rebuildingalliance.org
Version:   ushahidi-Ushahidi_Web-2.5-0-g878d194.zip

We're added an overlay in "Other Layers" to show the boundaries of Congressional Districts -- sadly that  overlay is on top, above the reports shown in the Category Filter.  Can you help us add code to put the Category Filter above Other Layers?

To properly render report layers, we patched
ushahidi-Ushahidi_Web-f4df525
application/helpers/map.php:layers_array() to return a string in which
the first entry was the (default) base layer, followed by KML
overlays, and then by "other" layers.  However, this same patch, when
applied to ushahidi-Ushahidi_Web-2.5-0-g878d194.zip, does not work —
the reports are incorrectly rendered below the KML overlay.

```
 In summary, we need a method to render layers in the following
```

order, from bottom to top:

```
(1) (Default) base layer
(2) KML overlays
(3) Other (e.g., report) overlays
```
